benches/src/bin/collect.rs

```diff
@@ -596,7 +596,108 @@ fn slope(x_y: Vec<(u64, u64)>) -> f64 {
         .map(|(x, y)| (*x as f64 - avg_x) * (*y as f64 - avg_y))
         .sum();
     let sq_x: f64 = x_y.iter().map(|(x, _)| (*x as f64 - avg_x).powi(2)).sum();
-    sum_x_y / sq_x
+    sq_x / sum_x_y
```

Comment:
I don't think this is correct for the least squares method of linear regression, it should be `sum_x_y / sq_x`:
![image](https://github.com/FuelLabs/fuel-core/assets/794823/2ace71d1-65bf-4fc7-acbf-68c4b3e8afc8)
https://en.wikipedia.org/wiki/Simple_linear_regression#Fitting_the_regression_line

Reply:
Yes, but we need value for `y`, not `x`. And it is the same behavior as before(after `slope` we divide `1/slope(x_y)`

Reply:
Or is it inverted because our dep_per_unit is inverted? Might be worth putting a comment explaining why this deviates from the standard formula.

Reply:
Added a comment=)